tasks/libs/ciproviders/gitlab_api.py

```diff
@@ -336,6 +364,27 @@ def str_note() -> list[str]:
 
         return '\n'.join(res)
 
+    def iter_jobs(self, added=True, modified=True, removed=False):
+        """
+        Will iterate over all jobs in all files for the given states
+
+        Returns a tuple of (job_name, contents, state)
+
+        Note that the contents is the contents after modification or before removal
```

Comment:
This is not really clear, could you clarify?

Reply:
> Note that the contents of the job is the contents after modification if modified or before removal if removed